### PR TITLE
Some test improvements

### PR DESCRIPTION
**Pull request by spacecowboy**
Was enabling coverage on a new project and tried some copy paste from Lifelines and probably ran into the same problems you did (I am assuming you ran into the problems ;)

Also added some new Matploblib versions to the test matrix. (Btw, 2.0 is apparently in dev)

### CODE REVIEW

**Comment by spacecowboy** [comment]
[94.8% coverage](https://coveralls.io/builds/5210552)

File: .travis.yml [hunk 1]
```diff
@@ -71,10 +75,10 @@ install:
     fi
 
   # Build in place so we can run tests
-  - pip install coveralls pytest-cov
+  - pip install python-coveralls pytest-cov
```
Comment (CamDavidsonPilon) [inline .travis.yml]: yessss

**Comment by CamDavidsonPilon** [comment]
:ok_hand: thanks for fixing this @spacecowboy!

**Comment by CamDavidsonPilon** [comment]
:ship:, closes https://github.com/CamDavidsonPilon/lifelines/issues/180

**Pull request merged**